sdk/python/feast/registry.py

```diff
@@ -71,7 +69,7 @@ def __init__(self, registry_path: str, repo_path: Path, cache_ttl: timedelta):
 
     def _initialize_registry(self):
         """Explicitly forces the initialization of a registry"""
-        self._registry_store.update_registry_proto()
+        self._registry_store.update_registry_proto(updater=None)
```

Comment:
nit: you don't need `updater=None` since you added the default

Reply:
Yea, this was a relic from my debugging. Good catch.